### ISSUE TITLE
Applying the model - hard sigmoid on the logits?

### ISSUE THREAD

**Issue by giodesp** [gmendes9/multilingual_va_prediction#2]
Hi both, 
Thanks for sharing this great tool!

I am wondering whether my application of the large model is correct as it is leading to a strange distribution of scores.
I am processing the logits with a hard sigmoid but not sure if this is the right approach?

```
from transformers import XLMRobertaForSequenceClassification, XLMRobertaTokenizer
def hard_sigmoid(logits):
    return torch.clamp((logits + 1) / 2, min=0, max=1)

model_path = ".../XLM-RoBERTa-large MSE"

model = XLMRobertaForSequenceClassification.from_pretrained(model_path)
tokenizer = XLMRobertaTokenizer.from_pretrained(model_path, local_files_only=True)

inputs = tokenizer(text, padding=True, truncation=True, max_length=128, return_tensors="pt")
outputs = model(**inputs)
logits = outputs.logits
scores = hard_sigmoid(logits).detach().numpy()
```

The scores seem to make sense but are yielding a strange distribution:
<img width="1772" alt="image" src="https://github.com/gmendes9/multilingual_va_prediction/assets/112628014/708204da-d2b9-4f1e-92c3-90cdae0ef52d">

Any help would be really appreciated. Thanks!

**Comment by giodesp** [comment]
In the end, I found that using a standard sigmoid led to a normal-ish distribution:
<img width="856" alt="image" src="https://github.com/gmendes9/multilingual_va_prediction/assets/112628014/73b66af1-7de9-43a3-a78f-51adc8fe7adf">

**Comment by NixonZ** [comment]
Thanks for sharing!
I am finding a similar issue, the hard sigmoid that the paper mentioned gives a strange distribution of the valence and arousal scores: most of the scores will be clipped at 0 and 1, as you are observing.

@gmendes9 should clarify what activation function they used for training so we can use it for inference.
